tonic/Cargo.toml

```diff
@@ -35,14 +35,15 @@ transport = [
 ]
 tls = ["transport", "tokio-rustls"]
 tls-roots = ["tls", "rustls-native-certs"]
+tls-dangerous = ["rustls", "webpki"]
```

Comment:
This should only have to enable `"rustls/dangerous_configuration"` then we leave it out of the default. So this is an opt-in feature that will only enable that specific rustls feature if you opt in.

Reply:
I didn't know feature flags were transitive, I'm looking into it.

Reply:
After thinking about it, I don't understand what you want me to do

Reply:
It should be possible to set `tls-dangerous` to be like:

```toml
tls-dangerous = ["rustls/dangerous-config", "webpki"]
```

Since, rustls is already included with `tokio-rustls` this feature set should work.

Let me know if that makes sense!